grid/project-management/tasks/maps/__snapshots__/asana.test.ts.snap

```diff
@@ -7,4 +7,4 @@ Ok {
     "url": "https://app.asana.com/0/1202159846567214/1202252122545982",
   },
 }
-`;
+`;
```

Comment:
Is this supposed to happen? I don't think you should have this PR touch this file at all.